core/domain/classifier_services.py

```diff
@@ -356,3 +356,82 @@ def delete_classifier_training_job(job_id):
         classifier_models.ClassifierTrainingJobModel.get(job_id))
     if classifier_training_job_model is not None:
         classifier_training_job_model.delete()
+
+
+def get_classifier_exploration_mapping(
```

Comment:
@anmolshkl Thanks for pointing that out. I initially made it for something else, but yeah. I've removed it.